src/EventListener/ExceptionListener.php

```diff
@@ -96,10 +100,6 @@ public function onConsoleError(ConsoleErrorEvent $event): void
             }
         }
 
-        if ($error instanceof \TypeError && str_contains($error->getMessage(), 'AcquiaCloudApi\Response')) {
```

Comment:
Type errors should not be tolerated: https://github.com/typhonius/acquia-php-sdk-v2/pull/477